requirements.txt

```diff
@@ -9,3 +9,4 @@ pillow
 email_validator
 requests
 pillow
+bleach
```

Comment:
Oh, I was initially using flask_ckeditor, but when I switched to version 5 I just went with the JavaScript CDN script. 
But I forgot to remove one of the unused imports I used for flask_ckeditor in one of the forms, so that might be causing issues. I'll push a change, so let me know if it fixes it.

Reply:
Ah, that makes sense. When I tried to run it, it complained about missing flask_ckeditor.